keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -234,7 +233,7 @@ you're proposing, but should not include things like API designs or
 implementation.  The "Design Details" section below is for the real
 nitty-gritty.
 -->
-#### Terminology
+### Terminology
```

Comment:
I'd rather leave this as it was - terminology is part of the preamble and doesn't need the same weight as the other sections.

Reply:
done: reverted level and squashed commits.
Thanks for the quick turnaround!